libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/api/ElasticsearchEntitlementChecker.java

```diff
@@ -66,10 +66,10 @@ private static Module requestingModule(Class<?> callerClass) {
     }
 
     private static boolean isTriviallyAllowed(Module requestingModule) {
-        if (isActive == false) {
-            logger.debug("Trivially allowed: entitlements are inactive");
-            return true;
-        }
+        // if (isActive == false) {
```

Comment:
@prdoyle should we get rid of all `isActive` stuff? Seems unused now.
Or do we want to keep it and call `activate()` during initialization?

Reply:
🔪🔪🔪

Reply:
If we see no need for it (I don't) let's delete it. This thing already has enough moving parts.